CHANGELOG.md

```diff
@@ -11,6 +11,10 @@ Recommendation: for ease of reading, use the following order:
 - Fixed
 -->
 
+## [Unreleased]
+### Fixed
+- GQL: datasets/createEmpty returns alias prefixed with account name 
```

Comment:
Didn't know there were anchors with issue link. Thanks for the hint!
I found the only place in tests: https://github.com/kamu-data/kamu-cli/blob/62ac6732c094ddbc8c2945ece3bfc01be3453d8d/src/e2e/app/cli/common/src/kamu_api_server_client_ext.rs#L496
Is it correct assumption that alias format is `account/dataset-name`, where `account/` prefix is optional, and present only in case of multi-tenant workspace? Otherwise, I don't see options to construct correct `DatasetAlias`.

Reply:
There's also issue mentioning in main code: https://github.com/kamu-data/kamu-cli/blob/62ac6732c094ddbc8c2945ece3bfc01be3453d8d/src/adapter/graphql/src/mutations/datasets_mut.rs#L112

Should we do anything more, or both TODO's can be removed from that place?

Reply:
tests were updated